requirements_rtfd.txt

```diff
@@ -1,4 +1,3 @@
-setuptools
-setuptools_scm
+scikit-build
```

Comment:
Looks like `requeriments.txt` makes more sense as a name for dependencies.  Could you rename it here?

Reply:
Good idea.

I moved `scikit-build` out of `requirements_rtfd.txt` into `requirements-dev.txt` since it is a build requirement, if that is what you would like. I can tweak further as desired.